### ISSUE TITLE
Todo: examine fast autoaugment

### ISSUE THREAD

**Issue by andrewjong** [kourai-research/pytorch-boilerplate#3]
Fast auto augment for auto augmenting your training data.

**Comment by SBoulanger** [comment]
Is this a library or a general concept (autoaugment)?
How is this different to what you would add to the transforms in a DataLoader?

**Comment by andrewjong** [comment]
AutoAugment is an algorithm for learning data augmentation. It's shown to work better than traditional augmentation. Fast AutoAugment is the fast version

**Comment by SBoulanger** [comment]
Took a look at Fast AutoAugment README.md. It takes 450 GPU hours to search for an augmentation strategy that is model (ResNet50) AND dataset (Reduced ImageNet) specific. After the search, it is all for a 1% gain in top1 accuracy and .3 percent gain in top 5. This seems too granular and expensive of a process to add to this generic repo. Could be an optional feature we add in the future.
